admin-manual/maintenance/cli-tools.rst

```diff
@@ -2728,6 +2728,48 @@ terminal.
 
 :ref:`Back to top <maintenance-cli-tools>`
 
+Find data integrity issues
+==========================
+
+Tries to find data integrity issues in the AtoM database, and attempts to repair
```

Comment:
Optional, but: could be good to provide a very brief, high-level overview of what data corruption in AtoM is and why it can occur (e.g. long-running operations timing out before completing, leaving incomplete data rows across the AtoM db).... just a thought. 

There is some similar text that could be borrowed from here: 

* https://www.accesstomemory.org/docs/latest/admin-manual/maintenance/troubleshooting/#dealing-with-data-corruption